kview/src/main/java/dev/ikm/komet/kview/mvvm/view/pattern/PropertiesController.java

```diff
@@ -317,10 +319,17 @@ private void setupDescriptionNamePane(EvtType eventType) {
             });
         } else if (eventType == SHOW_EDIT_FQN) {
             descrConfig.updateViewModel("descrNameViewModel", (descrNameViewModel) -> {
```

Comment:
Can we use the new features of Cognitive(mvvm library) to update a view model?
In line 210 Cognitive(mvvm) now allows you to obtain the view model from the JFXNode **after** it was created (via make()). 

- Functional style: https://github.com/carldea/cognitive/wiki#jfxnodes-updateviewmodel
- Imperative style: https://github.com/carldea/cognitive/wiki#jfxnodes-getviewmodel